packages/paste-core/utilities/media-object/src/index.tsx

```diff
@@ -1,50 +1,45 @@
-import styled from '@emotion/styled';
-import {space, Theme} from 'styled-system';
-import {themeGet} from '@styled-system/theme-get';
-import {SpaceProps} from '@twilio-paste/style-props';
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Space} from '@twilio-paste/style-props';
 
-export interface MediaObjectProps extends Theme {
-  isCentered?: boolean;
+export interface MediaObjectProps {
+  verticalAlign?: 'center' | 'top';
 }
 
-const MediaObject = styled.div<MediaObjectProps>`
-  align-items: ${(props: MediaObjectProps) => (props.isCentered ? 'center' : 'flex-start')};
-  display: flex;
-`;
+const MediaObject: React.FC<MediaObjectProps> = ({children, verticalAlign = 'top'}) => {
+  return (
+    <Box display="flex" alignItems={verticalAlign === 'top' ? 'flex-start' : 'center'}>
+      {children}
+    </Box>
+  );
+};
+MediaObject.displayName = 'MediaObject';
 
-export interface MediaFigureProps extends Theme {
+export interface MediaFigureProps {
   align?: 'start' | 'end';
-  spacing?: 'space20' | 'space30';
+  spacing?: Space;
 }
-
-const getMargin = (props: MediaFigureProps): string => {
-  const direction = props.align === 'end' ? 'left' : 'right';
-  const spacing = themeGet(`space.${props.spacing}`)(props);
-  return `margin-${direction}: ${spacing}`;
+const MediaFigure: React.FC<MediaFigureProps> = ({children, align, spacing}) => {
+  return (
+    <Box
+      display="flex"
+      flexShrink={0}
+      marginLeft={align === 'end' ? spacing : undefined}
```

Comment:
I don't think the Flex component currently allows margins, which may be an oversight after looking at this..